rdm/init_files/documents/test_record.md

```diff
@@ -40,6 +47,13 @@ It is ok if some tests do not trace to any particular requirements, however all
 {% endfor %}
 
 ## Integration Tests
+[[These are the results of automated integration testing 62304:5.6.3 62304:5.6.4 62304:9.8.a]]
```

Comment:
I think the references to 5.6.3 and 5.6.4 should go somewhere in the software plan.

Reply:
Fixed.